style.css

```diff
@@ -300,10 +332,135 @@ ul.media-links {
 }
 
 .app-contact form {
-  margin-top: 45px;
+  margin-top: 30px;
   gap: 20px;
 }
 
 .app-contact .contact-button {
   padding: 12px 16px;
 }
+
+@media (768px <= width) {
+  /* ----------- DESK-APP-BAR ----------- */
+
```

Comment:
Nice work for implementing almost all the project requirements to near perfection. :1st_place_medal:. At a point I was lost as to the difference between the figma design and your design. Keep it up. :congratulations: 

However, I realised that at certain media points/screen sizes, the picture looks smaller than the other side in the project card leaving empty white spaces on top and bottom of the image as shown below.  Kindly fix that in order to make your project more responsive thereby providing a smooth user experience. 

Your design             |  Figma design
:-------------------------:|:-------------------------:
![Screenshot from 2022-09-29 18-27-16](https://user-images.githubusercontent.com/65233860/193114826-3d319fa8-d2f7-4cd5-8a19-ee21273caefd.png)  |  ![Screenshot from 2022-09-29 18-44-43](https://user-images.githubusercontent.com/65233860/193116411-598e18ee-a622-4c1c-b5a9-464de4e1f122.png)